Rethinking_2/Chp_06.ipynb

```diff
@@ -0,0 +1,3109 @@
+{
```

Comment:
<p>You can also set <code>az.rcParams["stats.credible_interval"] = 0.89</code> here, so that you can get rid of all the <code>credible_interval=.89</code> below</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>